plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -402,7 +402,7 @@ func ClusterRoles() []rbacv1.ClusterRole {
 				eventsRule(),
 				rbacv1helpers.NewRule("create").Groups(legacyGroup).Resources("endpoints", "secrets", "serviceaccounts").RuleOrDie(),
 				rbacv1helpers.NewRule("delete").Groups(legacyGroup).Resources("secrets").RuleOrDie(),
-				rbacv1helpers.NewRule("get").Groups(legacyGroup).Resources("endpoints", "namespaces", "secrets", "serviceaccounts").RuleOrDie(),
+				rbacv1helpers.NewRule("get").Groups(legacyGroup).Resources("endpoints", "namespaces", "secrets", "serviceaccounts", "configmaps").RuleOrDie(),
```

Comment:
The existing entries appear to be in alphabetical order, likely deliberately. Can we preserve that rather than adding the new resource at the end?

Reply:
> The existing entries appear to be in alphabetical order, likely deliberately. Can we preserve that rather than adding the new resource at the end?

Original author here.  That was a complete coincidence.  Most in this file aren't alphabetical.  I don't really care one way or the other.